ownCloud/Client/SortBar.swift

```diff
@@ -163,7 +163,7 @@ class SortBar: UIView, Themeable {
 			stackView.topAnchor.constraint(equalTo: topAnchor),
 			stackView.bottomAnchor.constraint(equalTo: bottomAnchor),
 			stackView.leftAnchor.constraint(equalTo: safeAreaLayoutGuide.leftAnchor, constant: 20),
-			stackView.rightAnchor.constraint(equalTo: safeAreaLayoutGuide.rightAnchor, constant: -20)
+			stackView.rightAnchor.constraint(equalTo: safeAreaLayoutGuide.rightAnchor, constant: -50)
```

Comment:
@hosy I am not a friend of magic numbers spread over the code. Would rather prefer using let constants. But may this is something we could do later.

Reply:
> magic numbers

👎

Reply:
That's correct. There a many magic numbers to fix in this implementation. This was just a quick fix but I can do this now.

Reply:
@hosy @michaelstingl I think this is not only UI class which uses magic numbers. We could create a separate task to move spacing, margin values into a separate file defining these constants.